test/e2e/kubevirt.go

```
@@ -13,25 +13,17 @@
 	"sync"
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	iputils "github.com/containernetworking/plugins/pkg/ip"
+	butaneconfig "github.com/coreos/butane/config"
+	butanecommon "github.com/coreos/butane/config/common"
+	ipamclaimsv1alpha1 "github.com/k8snetworkplumbingwg/ipamclaims/pkg/crd/ipamclaims/v1alpha1"
+	nadapi "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	nadv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```

golangci-lint (1.64.8)

[warning] duplicated-imports: Package "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1" already imported

(revive)